aip_common_sensor_launch/launch/nebula_node_container.launch.py

```diff
@@ -441,6 +443,13 @@ def add_launch_arg(name: str, default_value=None, description=None):
     add_launch_arg("advanced_diagnostics", "false")
     add_launch_arg("use_multithread", "False", "use multithread")
     add_launch_arg("use_intra_process", "False", "use ROS 2 component container communication")
+    add_launch_arg("hires_mode", "false")
```

Comment:
This change will conflict with https://github.com/tier4/aip_launcher/pull/497

Reply:
<img width="1006" height="1006" alt="image" src="https://github.com/user-attachments/assets/4f0c666b-3b14-4ee9-9f1d-7ee01297f29b" />